boot/boot_images.cmake

```diff
@@ -0,0 +1,165 @@
+## efisys.bin
+
+# EFI platform ID, used in environ/CMakelists.txt for bootmgfw filename naming also.
+if(ARCH STREQUAL "amd64")
```

Comment:
Would it make sense to do this in `/CMakeLists.txt` next to where `WINARCH` is defined.

Reply:
It would, maybe, if we needed this identifier elsewhere than just in this very precise place.